### ISSUE TITLE
issue with loading

### ISSUE THREAD

**Issue by synapse25** [knod/read_plugin#14]
When you go to a site and its still loading and you click on the read entire page button, it doesn't do anything. You have to wait the page to completely finish loading and press it again. We could fix this by at least making it work automatically as soon as the page finishes loading.

**Comment by knod** [comment]
I believe this is not possible with a Chrome extension. I suspect it's not possible at all. We can try to see if there's anything to do about it if/when we convert the project to a bookmarklet.

**Comment by ghost** [comment]
Others (who have knowledge) have told me that this is possible - just mentioning it to clarify it. Will try to get more details on how exactly it is possible.

**Comment by knod** [comment]
Yeah, think I found some chrome events that may allow this to happen. Forgot to update the issue.
